graphql/data/schema.js

```diff
@@ -166,6 +170,9 @@ const { nodeInterface, nodeField } = nodeDefinitions(
     if (type === VIDEO_AD_LOG) {
       return VideoAdLogModel.get(context.user, id)
     }
+    if (type === CAUSE) {
+      return getCause(context.user, context.user.id)
```

Comment:
We will still want a node to resolve based on its global ID here. The question is more: do we need a new node definition for the object? If not—which is the case for Causes at the moment—we don't need this at all. But if we did want a node definition, we'd want stable IDs that resolve to every instance.